Calculator.Tests/CalculatorTests.cs

```diff
@@ -0,0 +1,33 @@
+using NUnit.Framework;
+
+namespace Calculator.Tests
+{
+    public class CalculatorTests
+    {
+        [SetUp]
+        public void Setup()
+        {
+            
+        }
+
+        [Test]
+        public void DoNothing()
```

Comment:
After you start writing your test methods (beginning with the simplest one), this test can be removed.